src/Nethermind/Nethermind.Evm/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -562,6 +562,12 @@ protected virtual void PayFees(Transaction tx, BlockHeader header, IReleaseSpec
                     UInt256 fees = (UInt256)spentGas * premiumPerGas;
                     UInt256 burntFees = !tx.IsFree() ? (UInt256)spentGas * header.BaseFeePerGas : 0;
 
+                    if (tx.SupportsBlobs)
+                    {
+                        BlobGasCalculator.TryCalculateBlobGasFee(header, tx, out UInt256 blobGasFee);
+                        burntFees += blobGasFee;
```

Comment:
Is this something we want to report in ReportFees? Maybe yes

Reply:
I think it does since ReportFees reports burntFees, which now includes the blob gas fees. Do you think we should give the breakdown in ReportFees of specifically blob gas as well?